hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/staking/StakeRewardCalculatorImpl.java

```diff
@@ -36,15 +38,19 @@
 @Singleton
 public class StakeRewardCalculatorImpl implements StakeRewardCalculator {
     private static final Logger logger = LogManager.getLogger(StakeRewardCalculatorImpl.class);
+
     private final StakePeriodManager stakePeriodManager;
+    private final InstantSource instantSource;
 
     /**
      * Default constructor for injection.
      * @param stakePeriodManager the stake period manager
      */
     @Inject
-    public StakeRewardCalculatorImpl(@NonNull final StakePeriodManager stakePeriodManager) {
+    public StakeRewardCalculatorImpl(
+            @NonNull final StakePeriodManager stakePeriodManager, @NonNull final InstantSource instantSource) {
         this.stakePeriodManager = stakePeriodManager;
+        this.instantSource = Objects.requireNonNull(instantSource);
```

Comment:
Do we need this here ?

Reply:
Ah no Neeha, will remove in future PR. 🙏